apps/addons/models.py

```diff
@@ -373,9 +377,11 @@ class Addon(amo.models.OnChangeMixin, amo.models.ModelBase):
     # mistake. You thus want the CLASS of the first one to be filtered by
     # default.
     # We don't control the instantiation, but AddonManager sets include_deleted
-    # to False by default, so filtering is enabled by default. This is also why
-    # it's not repeated for 'objects' below.
+    # and include_unlisted to False by default, so filtering is enabled by
+    # default. This is also why it's not repeated for 'objects' below.
+    unfiltered = AddonManager(include_deleted=True, include_unlisted=True)
```

Comment:
Do we need both `unfiltered` and `with_deleted` ? Does `with_deleted` have an use case if `unfiltered` exists ?